Include/internal/pycore_pystate.h

```diff
@@ -19,23 +19,27 @@ extern "C" {
 // interpreter at the same time. Only the "bound" thread may perform the
 // transitions between "attached" and "detached" on its own PyThreadState.
 //
-// The "gc" state is used to implement stop-the-world pauses, such as for
-// cyclic garbage collection. It is only used in `--disable-gil` builds. It is
-// similar to the "detached" state, but only the thread performing a
-// stop-the-world pause may transition threads between the "detached" and "gc"
-// states. A thread trying to "attach" from the "gc" state will block until
-// it is transitioned back to "detached" when the stop-the-world pause is
-// complete.
+// The "suspended" state is used to implement stop-the-world pauses, such as
+// for cyclic garbage collection. It is only used in `--disable-gil` builds. It
+// is similar to the "detached" state, but only the thread performing a
+// stop-the-world pause may transition a thread from the "suspended" state back
+// to the "detached" state. A thread trying to "attach" from the "suspended"
+// state will block until it is transitioned back to "detached" when the
+// stop-the-world pause is complete.
 //
 // State transition diagram:
 //
 //            (bound thread)        (stop-the-world thread)
-// [attached]       <->       [detached]       <->       [gc]
+// [attached]       <->       [detached]       <->       [suspended]
+//   +                                                        ^
+//   +--------------------------------------------------------+
+//                          (bound thread)
```

Comment:
FWIW, the text above makes it sound like thread a thread can go from "suspended" straight to "attached".

Reply:
Also, `(bound thread)` shows up at the top of the diagram and at the bottom.  I'm not sure how to interpret that.